examples/word-count/Makefile

```diff
@@ -3,7 +3,7 @@
 run:
 	@test -e output || mkdir output
 	rm -f output/*
-	../../bin/slurm-pipeline.py -s specification.json --scriptArgs texts/*.txt > output/status.json
+	env PYTHONPATH=../../bin/slurm-pipeline.py -s specification.json --scriptArgs texts/*.txt > output/status.json
```

Comment:
This needs to be

```sh
env PYTHONPATH=../.. ../../bin/slurm-pipeline.py -s specification.json --scriptArgs texts/*.txt > output/status.json
```
So you put `../..` into the Python path, and you then run `../../bin/slurm-pipeline.py`.

Can you also do this in the `Makefile` in the other example directories too?

Reply:
Ah yes, I was wondering if that is correct, but I found the double ../.. confusing - if the PYTHONPATH looks in ../.., why the need to put it in front of the bin/slurm-pipeline.py? 
Yes I'll do the same in the other directories.